src/main/kotlin/fi/metatavu/timebank/api/keycloak/KeycloakController.kt

```diff
@@ -48,6 +49,44 @@ class KeycloakController {
         }
     }
 
+    /**
+     * Gets unspentVacations attribute for Person
+     * If not set will set and return default value of -1
+     *
+     * @param user UserRepresentation
+     * @return Int unspentVacations
+     */
+    fun getUsersUnspentVacationDays(user: UserRepresentation): Int {
+        return try {
+            user.attributes["unspentVacations"]!!.first()!!.toInt()
+        } catch (e: Exception){
+            updateUsersUnspentVacationDays(user, -1)
+            -1
+        }
+    }
+
+    /**
+     * Gets spentVacations attribute for Person
+     * If not set will set and return default value of -1
+     *
+     * @param user UserRepresentation
+     * @return Int spentVacations
+     */
+    fun getUsersSpentVacationDays(user: UserRepresentation): Int {
+        return try {
+            user.attributes["spentVacations"]!!.first()!!.toInt()
+        } catch (e: Exception){
+            updateUsersSpentVacationDays(user, -1)
```

Comment:
Same here as for the unspent vacation days, is there a calculation somewhere else for this- according to Ville, yes. So set that as the default value.